pkg/app/app.go

```diff
@@ -25,25 +25,25 @@ import (
 )
 
 func defaultFuncOrganizationSlug(engine workflow.Engine, config configuration.Configuration, logger *zerolog.Logger, apiClientFactory func(url string, client *http.Client) api.ApiClient) configuration.DefaultValueFunction {
-	callback := func(existingValue interface{}) interface{} {
+	callback := func(existingValue interface{}) (interface{}, error) {
 		client := engine.GetNetworkAccess().GetHttpClient()
 		url := config.GetString(configuration.API_URL)
 		apiClient := apiClientFactory(url, client)
 		orgId := config.GetString(configuration.ORGANIZATION)
 		if len(orgId) == 0 {
-			return existingValue
+			return existingValue, nil
 		}
 		slugName, err := apiClient.GetSlugFromOrgId(orgId)
 		if err != nil {
 			logger.Print("Failed to determine default value for \"ORGANIZATION_SLUG\":", err)
```

Comment:
Just a note, I tried to keep most of the returns untouched as not to alter behaviour, and I could always come back and add the error to the return if necessary. The returns that changed were because of the linter.